src/backend/distributed/utils/citus_clauses.c

```diff
@@ -254,6 +254,7 @@ ShouldEvaluateExpression(Expr *expression)
 		case T_RowCompareExpr:
 		case T_RelabelType:
 		case T_CoerceToDomain:
+		case T_NextValueExpr:
```

Comment:
Why is this necessary?

Reply:
It's needed for identity column value evaluation (which is a T_NextValueExpr).